tools/format_python_tools.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+VENV_DIR="pyformat"
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
```

Comment:
Why not just use `realpath` here instead of `cd/pwd`?

Reply:
I noticed this pattern in other shell scripts so I used it for consistency. I can update :)